administration/src/bp-modules/stores/StoresRequirementsText.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components'
+
+import { FILE_SIZE_LIMIT_MEGA_BYTES } from './StoresCSVInput'
+
+const RequirementsList = styled.ul`
+  text-align: left;
+  padding-left: 20px;
+`
+
+type ImportCardsRequirementsProps = {
+  header: string[]
+}
+
+const StoresRequirementsText = ({ header }: ImportCardsRequirementsProps) => {
+  return (
+    <RequirementsList>
+      <li>Maximale Dateigröße: {FILE_SIZE_LIMIT_MEGA_BYTES}MB</li>
+      <li>Dateiformat: CSV</li>
+      <li>Spaltenformat: {header.join(', ')}</li>
```

Comment:
```suggestion
      <li>Erforderliche Spalten:: {header.join(', ')}</li>
```
Nit-picky one, since it's not a format but a list of required columns